### PR TITLE
update api call

### PR DESCRIPTION
**Pull request by kpthatsme**
Fixes https://sentry.io/organizations/posthog/issues/2372018478/?referrer=slack

### CODE REVIEW

File: multi_tenancy/stripe.py [hunk 1]
```diff
@@ -146,7 +146,7 @@ def report_subscription_item_usage(subscription_id: str, billed_usage: int, time
     _init_stripe()
 
     subscription = get_subscription(subscription_id)
-    subscription_items = subscription.items.data
+    subscription_items = subscription.get('items', {}).get('data', [])
```
Comment (paolodamico) [inline multi_tenancy/stripe.py:149]: Were you able to test this with an actual API call? I'm worried that the `stripe-python` library does some parsing that results in the data being returned differently. Tests also need updating, let me know if I can help.

Comment (kpthatsme) [reply multi_tenancy/stripe.py:149]: Yea I had written a local script to test this:
```   
def t():
    import stripe
    stripe.api_key = "redacted"
    sub = stripe.Subscription.retrieve("redacted")
    sub_items = sub.get('items', {}).get('data', [])
    for item in sub_items:
        print(item.get('id'))
        print(item.get('price').get('recurring').get('usage_type'))

```
And it worked.

Do cloud tests run for you locally? Struggling to get that running to fix the test

Comment (paolodamico) [reply multi_tenancy/stripe.py:149]: Okay cool, I'll update the tests. Haven't tried recently, but can just fix it with the CI.

**Review by paolodamico — APPROVED** [review]
🚀

**Pull request merged**